tests/components/composer.test.ts

```diff
@@ -1214,6 +1214,13 @@ describe("composer", () => {
     expect(fixture.querySelectorAll(".o-grid div.o-composer")).toHaveLength(0);
   });
 
+  test("The composer helper should be closed on toggle topbar context menu", async () => {
```

Comment:
this test would be better placed in the file `top_bar.test.ts` as it tests the code of top_bar, not just the composer ;-)

Reply:
@rrahir Updated requested changes please check, though i guess the scope of the issue is in both grid and topbar so if you suggest we can keep the test case for both in top_bar.test.ts and grid.test.ts.

Reply:
How is this affecting the grid?

Reply:
formula-assitant to be closed if opened in grid-composer as well.

Reply:
that seems to be a bit out of the scope of the task, isn't it?  Are you sure it's not tested yet? if not then yes it would make sense to test that the formula assistant closes in the grid as well. If you add this, be sure to update the commit message accordingly

Reply:
Okay i've added the test and updated commit message as well.